pint_server/models.py

```diff
@@ -42,6 +42,11 @@ class ServerType(enum.Enum):
 
 
 class PintBase(object):
+    @property
+    def tablename(self):
```

Comment:
where's this being used?

Reply:
In pint-data-manager as a generic property to determine the table name without having to access a double underscore attribute outside the pint_server code base ;-)